README.md

```diff
@@ -168,6 +177,13 @@ nginx       11  0.0  0.3  31988  1968 ?        S    19:33   0:00 nginx: worker p
 nginx       65  0.6  9.1 111584 45884 ?        S    19:33   0:06 controller-agent
 ```
 
+If you are running NGINX Con container with NAP (NGINX App Protect) then you should also see NAP-specific processes:
```

Comment:
If your container includes NAP (NGINX App Protect) then you should also see NAP-specific processes: